core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -2743,33 +2743,9 @@ object Stream extends StreamLowPriority {
           .emit(r)
           .covary[F]
           .map(o => (token, o))
-          .get[F, (Token, R)]
-          .transformWith(bracketFinalizer(token))
+          .get[F, (Token, R)] >> Algebra.release(token)
```

Comment:
What about the `Result.Interrupted` case? Why is it not relevant anymore? Or is it dead-code as well given that it's only `emit` and then `map`?

Reply:
That case was an identity -- it indeed gets hit but doesn't alter the result in any way.

Reply:
Ah, ofc, it's just propagating the same `Interrupted` node